content/docs/1.5.2/advanced-resources/rwx-workloads.md

```diff
@@ -31,22 +31,62 @@ It is necessary to meet the following requirements in order to use RWX volumes.
 4. One can change the Longhorn volume's access mode via the UI as long as the volume is not bound to a PVC.
 5. For a Longhorn volume that gets used by a RWX PVC, the volume access mode will be changed to RWX.
 
+## Configuring Volume Mount Options
+
+An RWX volume is accessible only when mounted via NFS. By default Longhorn uses NFS version 4.1 with the `softerr` mount option, a `timeo` value of "600", and a `retrans` value of "5".  
+
+If the NFS server becomes inaccessible, requests from NFS clients are retried according to the configured `retrans` value. Longer-duration events such as power outages and factors such as network partitions cause the requests to eventually fail.  An NFS error (`ETIMEDOUT` for the `softerr` mount option) is returned to the calling application and data loss may occur. If `softerr` is not supported, Longhorn automatically uses the `soft` mount option instead, which returns an `EIO` as the error.
+
+You can use specific mount options for new volumes. First, create a customized StorageClass with an `nfsOptions` parameter, and then create PVCs for RWX volumes using that specific StorageClass.
+
+Example:
+
+  ```yaml
+  kind: StorageClass
+  apiVersion: storage.k8s.io/v1
+  metadata:
+    name: longhorn-test
+  provisioner: driver.longhorn.io
+  allowVolumeExpansion: true
+  reclaimPolicy: Delete
+  volumeBindingMode: Immediate
+  parameters:
+    numberOfReplicas: "3"
+    staleReplicaTimeout: "2880"
+    fromBackup: ""
+    fsType: "ext4"
+    nfsOptions: "vers=4.1,noresvport,softerr,timeo=600,retrans=5"
+  ```
+
+> **Important:**
+> To create PVCs for RWX volumes using the sample StorageClass, replace the `nfsOptions` string with a customized comma-separated list of legal options.
+
+### Notes
+
+1. You must provide the complete set of options.
+
+2. Longhorn does not validate the `nfsOptions` string, so erroneous values and typographical errors are not flagged.  When the string is invalid, the mount is rejected by the NFS server and the volume is not created nor attached.
+
+3. In Longhorn v1.4.0 to 1.4.3 and v1.5.0 to v1.5.1, volumes within a share manager pod (specifically, in the `NodeStageVolume` step) are hard mounted by default by the Longhorn CSI plugin.  Hard mounting allows Longhorn to persistently retry sending NFS requests, ensuring that IOs do not fail even when the NFS server becomes inaccessible for some time. IOs resume seamlessly when the server regains connectivity or a replacement server is created.
+
+   This mechanism for guaranteeing data integrity, however, comes with some risk.  To maintain stability, the Linux kernel does not allow unmounting of a file system until all pending IOs are completed.  This is a concern because the system cannot shut down until all file systems are unmounted.  If the NFS server is unable to recover, the client nodes must undergo a forced reboot.
```

Comment:
```suggestion
   This mechanism for guaranteeing data integrity, however, comes with some risk. To maintain stability, the Linux kernel does not allow unmounting of a file system until all pending IOs are completed. This is a concern because the system cannot shut down until all file systems are unmounted. If the NFS server is unable to recover, the client nodes must undergo a forced reboot.
```